app/core/classes/class.Posts.inc.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+/**
+*
+*
+*
+* Posts Class
+*
+*
+*
+*/
+class Posts
+{
+
+    function __construct()
+    {
+        // Config
+        include_once 'core/database.php';
+
+        // Connect to Database
+        try {
```

Comment:
Should the { be on the line underneath? Just noticed that you were using that convention for function and catch keywords